app/controllers/pensions/ReliefAtSourcePensionsController.scala

```diff
@@ -0,0 +1,93 @@
+import forms.YesNoForm
+import models.User
+import models.mongo.PensionsCYAModel
+import models.pension.reliefs.PaymentsIntoPensionViewModel
+import play.api.data.Form
+import play.api.i18n.I18nSupport
+import play.api.mvc.{Action, AnyContent, MessagesControllerComponents}
+import services.PensionSessionService
+import uk.gov.hmrc.play.bootstrap.frontend.controller.FrontendController
+import utils.{Clock, SessionHelper}
+import views.html.pensions.ReliefAtSourcePensionsView
+
+import javax.inject.Inject
+import scala.concurrent.{ExecutionContext, Future}
+
+class ReliefAtSourcePensionsController @Inject()(implicit val cc: MessagesControllerComponents,
+                                                 auth: AuthorisedAction,
+                                                 inYearAction: InYearAction,
+                                                 rasPensionView: ReliefAtSourcePensionsView,
+                                                 appConfig: AppConfig,
+                                                 pensionSessionService: PensionSessionService,
+                                                 errorHandler: ErrorHandler,
+                                                 ec: ExecutionContext,
+                                                 clock: Clock) extends FrontendController(cc) with I18nSupport with SessionHelper {
+
+  def show(taxYear: Int): Action[AnyContent] = auth.async { implicit user =>
+    pensionSessionService.getPensionsSessionDataResult(taxYear) {
+      case Some(data) =>
+        data.pensions.paymentsIntoPension.rasPensionPaymentQuestion match {
+          case Some(value) => Future.successful(Ok(rasPensionView(yesNoForm.fill(value), taxYear)))
+          case None => Future.successful(Ok(rasPensionView(yesNoForm, taxYear)))
+        }
+      case _ => Future.successful(Ok(rasPensionView(yesNoForm, taxYear)))
+
+    }
+  }
+
+  def submit(taxYear: Int): Action[AnyContent] = auth.async { implicit user =>
+    yesNoForm.bindFromRequest().fold(
+      formWithErrors => Future.successful(BadRequest(rasPensionView(formWithErrors, taxYear))),
+      yesNo => {
+        pensionSessionService.getPensionsSessionDataResult(taxYear) { optData =>
+
+          val pensionsCya = optData.map(_.pensions).getOrElse(PensionsCYAModel(PaymentsIntoPensionViewModel()))
+          val viewModel = pensionsCya.paymentsIntoPension
+
+          val updatedCyaModel = {
+            if (yesNo) {
+              pensionsCya.copy(paymentsIntoPension = viewModel.copy(rasPensionPaymentQuestion = Some(true)))
+            } else {
+              pensionsCya.copy(paymentsIntoPension = viewModel.copy(rasPensionPaymentQuestion = Some(false), totalRASPaymentsAndTaxRelief = None))
+            }
+          }
+          pensionSessionService.createOrUpdateSessionData(
+            updatedCyaModel, taxYear, optData.exists(_.isPriorSubmission))(errorHandler.internalServerError()) {
+            if (yesNo) {
```

Comment:
There was a navigation omission on last PR (not to do with your PR) for tax relief not claimed  controller
It navigates to the overview page but the annuity page s actually available 
Would you mind changing the redirect to that page too instead of the overview?